src/libraries/Microsoft.PowerFx.Core/Functions/TexlFunction.cs

```diff
@@ -388,6 +388,8 @@ protected set
 
         public bool IsDeprecatedOrInternalFunction => this is IHasUnsupportedFunctions sdf && (sdf.IsDeprecated || sdf.IsInternal);
 
+        public virtual bool HasTypeArgs => false;

```

Comment:
Can you add a comment with an invariant saying that a function HasTypeArgs == true if and only if there is an argIndex such that func.HasIsType(argIndex) == true (and add a test to validate that)?

Reply:
done and added test